app/src/main/java/org/fossasia/openevent/general/utils/Utils.kt

```diff
@@ -77,6 +79,14 @@ object Utils {
         inputManager.hideSoftInputFromWindow(view.windowToken, InputMethodManager.SHOW_FORCED)
     }
 
+    fun setToolbar(activity: FragmentActivity?, title: String, hasUpEnabled: Boolean = true, show: Boolean = true) {
+        val thisActivity = activity as? AppCompatActivity
+        thisActivity?.supportActionBar?.title = title
+        thisActivity?.supportActionBar?.setDisplayHomeAsUpEnabled(hasUpEnabled)
```

Comment:
If the value for `show` is false. Then that logic makes no sense.

Reply:
Done